### PR TITLE
[Dev] [influxbd-retriever] ILPF generic route

### PR DESCRIPTION
**Pull request by rodrigoaasm**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
*#2347 

* **Other information**:

### CODE REVIEW

File: storage/time-series/influxdb/retriever/test/unit/GenericQueryService.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        table: 0,
+        _value: '_value',
+      },
+    ]);
+  });
+
+  test('should throw an error, when the query has a from operation', async () => {
+    expect.assertions(1);
+    try {
+      await genericQueryService.runQuery('default', 'from(bucket:"default")');
```
Comment (cfrancisco) [inline storage/time-series/influxdb/retriever/test/unit/GenericQueryService.test.js:35]: A very informative unit test!

**Review by cfrancisco — DISMISSED** [review]
The code splitting you did was great! GJ!!

Just to point out, maybe the DeviceDataRepository name isn't a representative name (DataQuery or DataQueryRepository makes more sense to me), but it's fine.

And I missed a unit test, or a readme that could show me a real example of a 'query' parameter in genericQueryService.

File: storage/time-series/influxdb/retriever/test/unit/GenericQueryService.test.js (hunk 1 above)
Comment (rodrigoaasm) [reply storage/time-series/influxdb/retriever/test/unit/GenericQueryService.test.js:35]: Thanks !!

**Review by luissajunior — DISMISSED** [review]

**Review by cfrancisco — APPROVED** [review]
Great job!

**Review by luissajunior — APPROVED** [review]

**Pull request merged**